test/hotspot/jtreg/vmTestbase/nsk/share/jdi/Debugee.java

```diff
@@ -461,7 +460,7 @@ public Event waitingEvent(EventRequest request, long timeout)
 
             EventSet eventSet = eventQueue.remove(timeLeft);
             if (eventSet == null) {
-                continue;
+                return null;
```

Comment:
This does not match the comment at the line 438:
```
     * Wait for the requested event and skip other events.
```

Reply:
What is implied here is waiting for "timeout" ms, not indefinitely. Nothing has changed in that regard as the method was always intended to return after the timeout expires.

Reply:
Okay, thanks.